js/index.js

```diff
@@ -40,3 +40,54 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Update header img
+let headerImg = document.getElementById('cta-img');
+headerImg.setAttribute('src', siteContent["cta"]["img-src"]);
+
+// Update middle img
+let middleImg = document.getElementById('middle-img');
+middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"]);;
+
+// Add navbar
+let navContent = ['Services', 'Product', 'Vision', 'Features', 'About', 'Contact'];
+let navBar = document.querySelectorAll('a');
+navBar.forEach(function(e,index){ e.textContent = navContent[index]});
+
+// Made each element green
+navBar.forEach(e => e.style.color = "green");
+
+// Adding new elements to navbar
+let newNavItem1 = document.createElement('a');
+newNavItem1.textContent = 'Blog';
+newNavItem1.style.color = 'green';
+let newNavItem2 = document.createElement('a');
+newNavItem2.textContent = 'News';
+newNavItem2.style.color = 'green';
+document.querySelector('nav').appendChild(newNavItem1);
+document.querySelector('nav').prepend(newNavItem2);
+
+// Left section
+let ctaHeading = document.querySelector('.cta-text h1');
+ctaHeading.innerHTML = siteContent['cta']['h1'];
+let ctaButton = document.querySelector('button');
+ctaButton.textContent = siteContent['cta']['button'];
+
+// Main section
+let headersArray = ['Features', 'About', 'Services','Product','Vision'];
```

Comment:
remember that even though the assignment may seem counter intuitive the idea is to take the values of an object and apply them to html elements through JS dom manipulation. Creative way to bypass the object though!